src/main/java/org/ibp/api/rest/preset/PresetResource.java

```diff
@@ -47,6 +47,17 @@
 		return new ResponseEntity<>(presetDTO, HttpStatus.OK);
 	}
 
+	@RequestMapping(value = "/crops/{cropname}/programs/{programUUID}/presets/{presetId}", method = RequestMethod.PUT)
```

Comment:
conceptually this should be a PATCH instead of a PUT because the name cant be changed. Given that you have a validation in place to avoid updating the name and this is not a business resource I think it is ok to leave it as it is now